.gitignore

```diff
@@ -5,10 +5,20 @@ doc/tag/*
 # Ignore CMake build directory.
 build/
 
-#Ignore build artifacts
+# Ignore build artifacts
 *.o
 
 # Ignore code coverage artifacts
 *.gcda
 *.gcno
 *.gcov
+
+# Ignore CBMC proof artifacts
+**/cbmc/proofs/**/gotos
+**/cbmc/proofs/**/html
+**/cbmc/proofs/**/logs
+**/cbmc/proofs/**/TAGS-*
+**/cbmc/**/README.md
+**/cbmc/proofs/Makefile.common
+**/cbmc/proofs/Makefile-project-targets
+**/cbmc/proofs/Makefile-project-testing
```

Comment:
I don't think you can ignore these files.  If they are symbolic links to the templates repository, then we need the symbolic links.  If they are actual files replacing the symbolic links, then we need the actual files.

Reply:
Per offline conversation we shall check in the symbolic links.